### PR TITLE
Fix: Add/remove aria-expanded attribute along with drawer visibility

### LINKED ISSUE #1: Drawer should apply 'aria-expanded' attribute
### Subject of the issue/enhancement/features
According to success criteria 4.1.2 Name, Role, Value. Buttons and links that unfold / collapse content on the page are to be coded with aria-expanded attribute.

### Your environment
* FW5.31.12

### Expected behaviour
Expanded drawer should have 'aria-expanded' attribute applied. Collapsing the drawer should remove this

### Actual behaviour
No such attribute exists

### PR DESCRIPTION
**Pull request by cahirodoherty-learningpool**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
Fixes adaptlearning/adapt-contrib-core#409 

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* aria-expanded attribute added/removed according to drawer visibility

### CODE REVIEW

**Review by eleanor-heath — APPROVED** [review]

**Review by ethan-lp — APPROVED** [review]

**Review by lemmyadams — APPROVED** [review]

File: js/DrawerView.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ class DrawerView extends Backbone.View {
   }
 
   showDrawer(emptyDrawer) {
-    this.$el.removeClass('u-display-none').removeAttr('aria-hidden');
+    this.$el.removeClass('u-display-none').removeAttr('aria-hidden').attr('aria-expanded');
```
Comment (oliverfoster) [inline js/DrawerView.js:117]:
```suggestion
    this.$el.removeClass('u-display-none').removeAttr('aria-hidden').attr('aria-expanded', 'true');
```

File: js/DrawerView.js [hunk 2]
```diff
@@ -205,7 +205,8 @@ class DrawerView extends Backbone.View {
       this.$el
         .css(direction)
         .addClass('u-display-none')
-        .attr('aria-hidden', 'true');
+        .attr('aria-hidden', 'true')
+        .removeAttr('aria-expanded');
```
Comment (oliverfoster) [inline js/DrawerView.js:209]:
```suggestion
        .attr('aria-hidden', 'true')
        .attr('aria-expanded', 'false');
```

**Review by oliverfoster — CHANGES_REQUESTED** [review]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

`aria-expanded="false"` is also meaningful

**Comment by oliverfoster** [comment]
Did you mean to update this extension? It's not in use at the moment.

**Comment by cahirodoherty-learningpool** [comment]
> Did you mean to update this extension? It's not in use at the moment. This behaviour is in adapt-contrib-core.

I am an idiot 🤣

**Pull request closed without merging**